Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM pytorchlightning/pytorch_lightning:base-cuda-py3.11-torch2.2-cuda12.1.0
```

Comment:
Since we're no longer using the pytorch lightning dockerfile, do you need to install pytorch lightning specifically?

Reply:
`pip install gReLU` takes care of lightning installation